packages/bpk-component-content-cards/src/ContentCard/ContentCard.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import BpkText from 'bpk-component-text';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './ContentCard.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type CardLayout = 'HORIZONTAL' | 'VERTICAL';
+
+type Props = {
+  card: {
+    imageLink?: string;
+    headline?: string;
+    description?: string;
+    ctaLink?: string;
+  };
+  layout: CardLayout;
+};
+
+const ContentCard = ({ card, layout }: Props) => (
+  <a
+    className={[
+      getClassName('bpk-content-cards__card-link'),
+      layout === 'HORIZONTAL'
+        ? getClassName('bpk-content-cards__card-horizontal')
+        : getClassName('bpk-content-cards__card-vertical'),
+    ].join(' ')}
+    href={card.ctaLink}
+    target="_blank"
+    rel="noreferrer"
+  >
+    <div className={getClassName('bpk-content-cards__image-aspect-ratio')}>
+      <img
+        className={getClassName('bpk-content-cards__image')}
+        alt=""
```

Comment:
We should probably expose this property rather than just setting a blank alt for assistive technologies?

Reply:
Good call. While we don't support `alt` text for the cards in sponsored campaigns, this component needs to be more generic and be able to handle it.

I think we actually might want to check with accessibility experts on what the experience should be for these cards.
Maybe changing the markup, keeping anchor as a heading, instead of whole card being a link (but keeping whole card clickable):
```tsx
<div role="list" class="bpk-content-cards"> // to not bother reseting built-in styles
  <div role="listitem">
    <img />
    <a href={href}>Heading</a>
    <p>description</p>
  </div>
</div>
```

Reply:
I'd say unless we have some real issues, I'd go with adding optional `alt` to card's `Props`, but keep current markup (maybe adding `list`/`listitem` though, it's a list of related content in the end)

Reply:
I would say let's regroup and discuss this with design and accessibility on this, just to make sure - I would avoid us waiting until we have issues and while we have the opportunity to make it accessible on first creation lets do it now as its harder to fix in the future once we are out

Reply:
I added an alt `prop` and the `list/listitem` markup, let me know if this is ok
<img width="510" alt="Screenshot 2022-08-09 at 16 15 27" src="https://user-images.githubusercontent.com/58072716/183672161-dd7b464b-5688-4ce7-9488-330097a5a075.png">

Reply:
in `/packages/bpk-component-content-cards/src/BpkContentCards.tsx`
<img width="526" alt="Screenshot 2022-08-09 at 17 42 13" src="https://user-images.githubusercontent.com/58072716/183697415-d750d730-8905-4344-a843-8b5d65daffc2.png">